tests/testerina-integration-test/src/test/java/org/ballerinalang/testerina/test/TestReportTest.java

```diff
@@ -66,13 +66,13 @@ public void testWarningForReportTools() throws BallerinaTestException, IOExcepti
     }
 
     @Test ()
-    public void testWarningForJacocoXMLFlag() throws BallerinaTestException {
-        String msg = "warning: ignoring --jacoco-xml flag since code coverage is not enabled";
-        String[] args = new String[]{"--jacoco-xml"};
+    public void testWarningForCoverageFormatFlag() throws BallerinaTestException {
+        String msg = "warning: ignoring --coverage-format flag since code coverage is not enabled";
+        String[] args = new String[]{"--coverage-format=xml"};
```

Comment:
Should there be a test for how we handle values other than "xml".

Reply:
Added with ff40fe5042f6c077b8f1da2948c3057e3350b6b2